plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -122,6 +123,24 @@ public void setLoginPageUri(String loginPageUri) {
     }
 
     /**
+     * Get session manager implementation class.
+     *
+     * @return session manager implementation class
+     */
+    public String getSessionManager() {
+        return sessionManager;
+    }
+
+    /**
+     * Set session manager implementation class.
+     *
+     * @param sessionManager session manager implementation class
+     */
+    public void setSessionManager(String sessionManager) {
```

Comment:
Test case added to validate `SessionManager`. Please check [AppConfigTest.testSessionManagerValidations()](https://github.com/vidurananayakkara/carbon-uuf-maven-tools/blob/5a26e623e56e6e95548588943cde5e4a9d445847/plugin/src/test/java/org/wso2/carbon/uuf/maven/bean/AppConfigTest.java#L63)